### PR TITLE
CEMS-1861 Dan/render html

### PR DESCRIPTION
**Pull request by danmcadams**
Update the DTS to render HTML as the default format, pdf is optional by adding ?format=pdf to the end of the request

### CODE REVIEW

File: src/DTS/Render/Render.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+
+namespace HomeCEU\DTS\Render;
+
+
+abstract class Render implements RenderInterface {
```
Comment (ccopto) [inline src/DTS/Render/Render.php:7]: any particular reason for this to be abstract if you don't have abstract methods?

File: src/DTS/Render/RenderHTML.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+namespace HomeCEU\DTS\Render;
+
+
+class RenderHTML extends Render {
+  public static function create(): RenderInterface {
+    return new static();
+  }
+
+  public function render(string $compiledTemplate, array $data = []): string {
+    $t = $this->renderToString($compiledTemplate, $data);
```
Comment (ccopto) [inline src/DTS/Render/RenderHTML.php:13]: what is t? text or template?

Comment (danmcadams) [reply src/DTS/Render/RenderHTML.php:13]: Technically... both

File: src/DTS/Render/Render.php (hunk 1 above)
Comment (danmcadams) [reply src/DTS/Render/Render.php:7]: I didn't want to allow use of this class, I wanted to the user to use the factory to create a Renderer, and I'd consider it "abstract" in the sense it implements an interface and in order to extend the class, you must implement those methods.

**Pull request merged**